py/setup.py

```diff
@@ -380,7 +386,7 @@ def run(self):
     long_description=long_description,
     ext_modules=ext_modules,
     install_requires=[
-        "torch==2.0.0",
+        "torch >=2.0.1,<2.1" if not LEGACY else "torch >=1.13.0,<2.0",
```

Comment:
Updated torch requirement to coincide with necessary versions for Legacy/Regular installs.